src/common/MouseeventListener.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { setActiveAnnotationIdAction } from '../store/annotations';
+
+export default function MouseeventListener(): null {
```

Comment:
This seems both too specific and too generic, since the only mouse event is `mousedown`. Should we call it something else, maybe `DeselectListener`?

Reply:
Sounds good, I'll change it to `DeselectListener`